.github/workflows/java-gradle-release.yaml

```diff
@@ -101,7 +100,8 @@ jobs:
 
       - name: Commit and push
         run: |
-          git commit -am "Changelog for version ${{ steps.evaluate-version.outputs.release-version }}" 
+          git add CHANGELOG.md
+          git commit -m "Changelog for version ${{ steps.evaluate-version.outputs.release-version }}"
```

Comment:
Does this really work when the changelog does not exist yet?

Reply:
Why shouldn't it? I add it to the index in the line before.

Reply:
But you cannot add files to the index if they don't exist right?

Reply:
No. But the file will exists after the previous steps